<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectEditDialog</class>
 <widget class="QDialog" name="ObjectEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="graphicPreviewLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>graphic preview</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Object type: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="objectTypeComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>X-coordinate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="xPosSpinBox">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Y-coordinate: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="yPosSpinBox">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Facing flag:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="leftFacingButton">
             <property name="text">
              <string>Left</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rightFacingButton">
             <property name="text">
              <string>Right</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="blinkingBox">
           <property name="text">
            <string>Flashing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="invisibilityBox">
           <property name="text">
            <string>Invisible</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Left-facing VRAM index:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="leftTileVRAMIndexBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1" rowspan="2">
          <widget class="QPushButton" name="vramAutofillButton">
           <property name="text">
            <string>Autofill</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Right-facing VRAM index: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="rightTileVRAMIndexBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="spawnParameterWidget">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="SpawnParameterStandardWidget" name="page"/>
         <widget class="SpawnParameterWarpWidget" name="page_2"/>
         <widget class="SpawnParameterManualWidget" name="page_3"/>
         <widget class="SpawnParameterMissileLauncherWidget" name="page_4"/>
         <widget class="SpawnParameterEnemyMineWidget" name="page_5"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpawnParameterStandardWidget</class>
   <extends>QWidget</extends>
   <header>spawnparameterstandardwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpawnParameterWarpWidget</class>
   <extends>QWidget</extends>
   <header>spawnparameterwarpwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpawnParameterManualWidget</class>
   <extends>QWidget</extends>
   <header>spawnparametermanualwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpawnParameterMissileLauncherWidget</class>
   <extends>QWidget</extends>
   <header>spawnparametermissilelauncherwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpawnParameterEnemyMineWidget</class>
   <extends>QWidget</extends>
   <header>spawnparameterenemyminewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ObjectEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ObjectEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
